website/components/demo-block.vue

```diff
@@ -61,15 +72,20 @@ import { nextTick } from 'vue'
 import hljs from 'highlight.js'
 import clipboardCopy from 'clipboard-copy'
 import compoLang from '../i18n/component.json'
-import { stripScript, stripStyle, stripTemplate } from '../util'
+import { stripScript, stripStyle, stripTemplate, stripSetup, removeSetup } from '../util'
 const version = '1.0.0' // element version
 const stripTemplateAndRemoveTemplate = code => {
-  const result = stripTemplate(code)
+  const result = removeSetup(stripTemplate(code))
   if (result.indexOf('<template>') === 0) {
     return result.replace(/^<template>/, '').replace(/<\/template>$/,'')
   }
   return result
 }
+const sanitizeHTML = str => {
+  var temp = document.createElement('div')
```

Comment:
Maybe using `createDocumentFragment` API will be better? This is not a mandatory change though.